flixel/input/gamepad/XboxButtonID.hx

```diff
@@ -125,16 +125,16 @@ class XboxButtonID
 	/**
 	 * Button IDs
 	 */
-	public static inline var A:Int = 10;
-	public static inline var B:Int = 11;
-	public static inline var X:Int = 12;
-	public static inline var Y:Int = 13;
-	public static inline var LB:Int = 8;
-	public static inline var RB:Int = 9;
-	public static inline var BACK:Int = 5;
-	public static inline var START:Int = 4;
-	public static inline var LEFT_ANALOG:Int = 6;
-	public static inline var RIGHT_ANALOG:Int = 7;
+	public static inline var A:Int = 0;
+	public static inline var B:Int = 1;
+	public static inline var X:Int = 2;
+	public static inline var Y:Int = 3;
+	public static inline var LB:Int = 4;
+	public static inline var RB:Int = 5;
+	public static inline var BACK:Int = 6;
+	public static inline var START:Int = 7;
+	public static inline var LEFT_ANALOG:Int = 8;
+	public static inline var RIGHT_ANALOG:Int = 9;
 	
 	public static inline var XBOX:Int = 14;
 	
```

Comment:
Ah, I hadn't even considered that with them not being buttons you only have the "pressed" information. Yeah, that's inconvenient...

If you can make that hack work consistently, I think we should go for that. Of course it would be even better if it was consistent with flash at lime / openfl level (which it used to be until that mentioned SDL change). I guess in that case we should remove the getters instead?